maker/src/bin/maker.rs

```diff
@@ -14,9 +14,11 @@ use rand::thread_rng;
 use rand::RngCore;
 use std::net::SocketAddr;
 use std::sync::Arc;
+use std::time::Duration;
 use tracing::metadata::LevelFilter;
 
 const ELECTRS_ORIGIN: &str = "tcp://localhost:50000";
+const PROCESS_TRADE_REQUESTS_INTERVAL: Duration = Duration::from_secs(30);
```

Comment:
Can you add a comment why `30` seconds were chosen? it's also worth mentioning if it was a random choice